scss/components/_dropdowns.scss

```diff
@@ -62,7 +78,7 @@
   overflow: hidden;
   text-align: left;
   top: u(3.4rem);
-  z-index: 999;
+  z-index: 10;
```

Comment:
My bad, since I originally wrote the 999, but let's use a z-index variable here.